src/main/java/frc/trigon/robot/subsystems/swerve/swervemodule/SwerveModule.java

```diff
@@ -44,21 +44,19 @@ public SwerveModule(int moduleID, double wheelDiameterMeters, double offsetRotat
     }
 
     /**
-     * Sets the target angle and speed of the module from a target module state for the angle, and a target current for the drive speed.
-     * This is used for PathPlanner to follow a path using feedforwards as well as chassisSpeeds.
+     * Sets the target angle and speed of the module from a {@link SwerveModuleState} for the angle, and a target current for the drive speed.
+     * This is used for PathPlanner to follow a path using feedforwards as well as driving from ChassisSpeeds.
      *
-     * @param swerveModuleState the target module state. Only uses the angles
+     * @param swerveModuleState the target state of the module. Only uses the angles
      * @param targetCurrent     the target current of the drive motor
      */
     public void setTargetState(SwerveModuleState swerveModuleState, double targetCurrent) {
```

Comment:
Why is it talking about chassis speeds if there is nothing written about chassis speeds in the method. I see it taking a swerveModuleState and not a ChassisSpeeds